src/api/general-services/pipeline-status.js

```diff
@@ -3,7 +3,7 @@ const AWS = require('../../utils/requireAWS');
 const ExperimentService = require('../route-services/experiment');
 const config = require('../../config');
 const getLogger = require('../../utils/getLogger');
-const pipelineConstants = require('./pipeline-manage/constants');
```

Comment:
I think that `pipelineConstants` was a better name

Reply:
changed